server/vulnerabilities/goval_dictionary/database.go

```diff
@@ -0,0 +1,71 @@
+package goval_dictionary
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	"github.com/fleetdm/fleet/v4/server/vulnerabilities/utils"
+	"strings"
+)
+
+type VulnDataSource interface {
```

Comment:
nit: it's more idiomatic to name an interface like this based on what it can do ie.`Evaluator`

Reply:
Dropping the interface entirely since we only have one implementation, and trimming the API a bit since we don't actually care about platform.